### PR TITLE
CDAP-15291 Migrate CDC plugins to new package name

### PR DESCRIPTION
**Pull request by mlozbin-cybervisiontech**
JIRA: https://issues.cask.co/browse/CDAP-15291

### CODE REVIEW

**Comment by mlozbin-cybervisiontech** [comment]
@albertshau @jayfjin could you, please, take a look?

File: src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java [hunk 1]
```diff
@@ -14,10 +14,10 @@
  * the License.
  */
 
-package co.cask.cdc.plugins.common;
+package io.cdap.plugin.cdc.common;
 
-import co.cask.cdap.api.data.format.StructuredRecord;
-import co.cask.cdap.api.data.schema.Schema;
+import io.cdap.cdap.api.data.format.StructuredRecord;
```
Comment (albertshau) [inline src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java]: I believe imports are supposed to be in alphabetical order

**Review by albertshau — COMMENTED** [review]
just one comment about order of imports

File: src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java (hunk 1 above)
Comment (mlozbin-cybervisiontech) [reply src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java]: yes, sure. What is wrong with current order?

Comment (albertshau) [reply src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java]: the io.cdap ones should come after the com.xyz ones

Comment (mlozbin-cybervisiontech) [reply src/main/java/io/cdap/plugin/cdc/common/AvroConverter.java]: I see. Fixed order in all files and checked by maven task.

**Review by albertshau — APPROVED** [review]
lgtm

**Pull request merged**